wp-content/plugins/tapestry/controller/class.tapestry-controller.php

```diff
@@ -55,6 +55,26 @@ public function updateTapestry($tapestry) {
         return $tapestry;
     }
 
+    /**
+     * Update Tapestry settings
+     * 
+     * @param  Object @settings
+     * @return Object Tapestry settings
+     */
+    public function updateTapestrySettings($settings) {
+        if (!$this->postId) {
+            return $this->_throwsError('INVALID_POST_ID');
+        }
+
+        $tapestry = get_post_meta($this->postId, 'tapestry', true);
+        $tapestry->settings = $settings;
+
+        $this->_updatePost($tapestry, 'tapestry');
```

Comment:
Is this doing anything? If not, could you please remove it? Thanks.

Reply:
Yes, it is used to update the post title of the post.

Reply:
Are you sure? The body of `_updatePost` function is setting the title of the post to `$post->title`, not `$post->settings->title` so either that function is wrong or this shouldn't be here.

Reply:
Oh sorry, I was looking at `TAPESTRY_NODE` .. I see that for `TAPESTRY` it is using settings.